resources/public/api/conf/1.0/application.yaml

```diff
@@ -193,9 +181,12 @@ paths:
           application/json:
             schema:
               allOf:
-              - $ref: '#/components/schemas/individualPayeTestSupportSaTaxAnnualSummaryRequest'
-            example:
-              scenario: HAPPY_PATH_1
+                - $ref: '#/components/schemas/individualPayeTestSupportSaEmploymentsAnnualSummaryRequest'
```

Comment:
This is show the schema for employments instead of tax. Please update the schema to `individualPayeTestSupportSaTaxAnnualSummaryRequest`